posta/utils.py

```diff
@@ -9,4 +10,4 @@ def imposta_destinatari_e_scrivi_messaggio(request, qs_destinatari=Persona.objec
     qs_destinatari = Persona.objects.filter(pk__in=qs_destinatari.values_list('id', flat=True))
     request.session["messaggio_destinatari"] = qs_destinatari
     request.session["messaggio_destinatari_timestamp"] = datetime.now()
-    return redirect("/posta/scrivi/")
+    return redirect("{}?da_elenco=1".format(reverse('posta-scrivi')))
```

Comment:
Chiarisco -- A meno che facciamo uno sforzo per il refactoring, sul quale sarei d'accordo, ma ehm, non e' giustificabile economicamente in questo momento. Consistenza > Correttezza, in questo caso.